configs/BUILD

```diff
@@ -32,3 +32,8 @@ genrule(
         ":configgen",
     ],
 )
+
+filegroup(
+    name = "google_com_proxy_port_0",
+    srcs = ["google_com_proxy_port_0.v2.yaml"],
```

Comment:
In general I would like keep/evolve configs/ and examples/ as something that end users can look at and learn from. Along those lines, if this is just used for testing, can we potentially move this into the test configs directory to avoid confusion?

Reply:
actually test/config does not currently have any test configurations.  Should I make a test/configs?  Or just put it under test/exe/testdata (which would be the Google custom, more or less).